### ISSUE TITLE
The mathematical formulas in word2vec tutorial are encoded in latex and can't be correctly displayed in MD viewers

### ISSUE THREAD

**Issue by lopezvit** [petewarden/tensorflow_makefile#8]
In the word2vec tutorial, there are pieces of text like:
$$ \begin{align} P(w_t | h) &= \text{softmax}(\text{score}(w_t, h)) \ &= \frac{\exp { \text{score}(w_t, h) } } {\sum_\text{Word w' in Vocab} \exp { \text{score}(w', h) } }. \end{align} $$
that are written in latex, but not converted into something that can be correctly visible in a Markdown document.

In this stackoverflow [question](https://stackoverflow.com/questions/35498525/latex-rendering-in-readme-md-on-github) they describe few ways to solve this problem.
